lib/maps.rb

```diff
@@ -38,8 +39,10 @@ def trackpoints_to_geodata_multiline(activity, trackpoints)
     end
 
     {'type' => 'MultiLineString',
+     'starttime' => Time.parse(move['startTime']).to_i,
+     'endtime' => Time.parse(move['startTime']).to_i,
      'coordinates' => coordinates,
-     'activity' => activity}
+     'activity' => move['activity']}
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.